BUILD.md

```diff
@@ -71,12 +72,12 @@
   `ssg-${PRODUCT}-guide-index.html`. For example `ssg-rhel7-guide-index.html`.
 
 4. Discover the following:
- * A pretty prose guide **in rhel6-guide.html** containing practical, actionable information for administrators
- * A concise spreadsheet representation (potentially useful as the basis for an SRTM document) in **rhel6-table-nistrefs-server.html**
+ * A pretty prose guide **in rhel7-guide.html** containing practical, actionable information for administrators
+ * A concise spreadsheet representation (potentially useful as the basis for an SRTM document) in **rhel7-table-nistrefs-server.html**
```

Comment:
The table referenced doesn't exist too, should we point to `tables/table-rhel7-nistrefs-stig-rhel7-disa.html`?